spec/user_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe 'DB table' do
+    it { is_expected.to have_db_column :id }
+    it { is_expected.to have_db_column :email }
+    it { is_expected.to have_db_column :created_at }
+    it { is_expected.to have_db_column :updated_at }
+    it { is_expected.to have_db_column :name }
+    it { is_expected.to have_db_column :encrypted_password }
+  end
+
+  describe 'Validations' do
+    it { is_expected.to validate_presence_of :name }
+    it { is_expected.to validate_length_of(:name).is_at_most(10)
```

Comment:
when it spans on multiple lines like this, I prefer to use a `do..end` block

```ruby
it do
  is_expected.to validate_length_of(:name).is_at_most(10).with_message(false)
end
```

Just my personal preference and also to stick to the rule of only using `{}` when the block can fit in one line :-)